packages/rum/NOTICE.txt

```diff
@@ -6,27 +6,6 @@ Copyright (c) 2017-present, Elasticsearch BV
 ---
 This product relies on elastic-apm-js-core
 
-MIT License
```

Comment:
@jahtalab Since we have the license in root repo for `js-core`, It wont be copied here. Should we do something here? Or is it good to be like this?

Reply:
I think we need to include the full text of the license with the artefacts we publish (e.g. to npm). Also, most packages I've seen have a license file in their package when installed from npm.

Reply:
When Lerna does publishing, the LICENSE is automatically included for all packages, So dont worry about that part. What i meant here is if we have to manually traverse the root directory for  copying LICENSE in Notice.txt

Reply:
LICENSE, README and NOTICE - all the three will be published while doing lerna publish for all the packages, 

The reason why elastic-apm-js-core LICENSE info missed in notice.txt is because we have a root level LICENSE and not package level, We either have to change script
1. to copy license in each package before running the `genNotice` script
2. Traverse the root directory for all deps under `/packages/`

Reply:
Ok, I misunderstood then! What about adding a parameter to the generateNotice with a list of internal package names and a path to the the license file in the root?

Reply:
Hmmm, feel like too much explicit info, We can grab the internal package names from the directory passed and then we can read the root level. I will go with option 2 and also take your consideration in to account.